FSW/src/headers/config.hpp

```diff
@@ -52,10 +61,18 @@ const int FILESIZE = BUFFERSIZE + TIMESTAMP_SIZE;
 // timing constants
 const unsigned long SAMPLE_PERIOD_MSEC = 1000 / (unsigned long)SAMPLING_RATE; // millisec, time between samples  
 const int WINDOW_LENGTH_MSEC = WINDOW_LENGTH_SEC * 1000; // milliseconds, length of science data
+const int SWEEP_TIMEOUT_MSEC = 1000; // milliseconds, time for ADCS to sweep optic across the sun
 
 // Events
 static RecurringEvent dataProcessEvent(SAMPLE_PERIOD_MSEC); // assuming that duration arg is ms
 static Event saveBufferEvent;
+static TimedEvent sunriseTimerEvent(WINDOW_LENGTH_MSEC);
+static TimedEvent sweepTimeoutEvent(SWEEP_TIMEOUT_MSEC);
+
+// FUTURE TEAMS: this event is invoked when the ADCS should switch its sweep direction

```

Comment:
This is hot

Reply:
Better get a damn thank you letter in the mail

Reply:
hahaha true